src/components/CommentsList.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import { ClipLoader } from "react-spinners";
+import * as api from "../utils/api";
+import CommentCard from "./CommentCard";
+import SingleArticle from "./SingleArticle";
+
+const CommentsList = () => {
+    const [comments, setComments] = useState();
+    const [isLoading, setIsLoading] = useState(false);
+    const [showComments, setShowComments] = useState(false);
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        setIsLoading(true);
+        api.getCommentsByArticleId(article_id).then((data) => {
+            setComments(data.comments);
+            setIsLoading(false);
+            setShowComments(true);
+        })
+    }, [article_id]);
+
+    if (isLoading) return <ClipLoader color="#36D7B7"/>
+
+    return (
+        <div className="Comments__List">
+                <SingleArticle showComments={showComments}/>
```

Comment:
Ideally SingleArticle would render the CommentList and not the other way around. 
it feels as though there's some sort of inverted parent-child relationship of components going on here